lib/services/azuresqldb/cmd-provision.js

```diff
@@ -230,10 +234,14 @@ var sqldbProvision = function (log, params) {
                                 callback(null);
                             }
                         });
-                    } else {  // sql server exists
+                    } else if (result.statusCode === HttpStatus.OK) {  // sql server exists
```

Comment:
Thanks for the suggestion. Will fix.
The reason why the logic like this is, the broker doesn't have the option to set privilege whether allow to create server before. At that time, we assumed the server is usually a new one. So it checks if statusCode === 404 first.